pkg/solana/client/client.go

```diff
@@ -182,6 +183,16 @@ func (c *Client) GetAccountInfoWithOpts(ctx context.Context, addr solana.PublicK
 	return c.rpc.GetAccountInfoWithOpts(ctx, addr, opts)
 }
 
+func (c *Client) GetMultipleAccountsWithOpts(ctx context.Context, accounts []solana.PublicKey, opts *rpc.GetMultipleAccountsOpts) (out *rpc.GetMultipleAccountsResult, err error) {
```

Comment:
Should we add a unit test for this method? Or is it covered by another component?

Reply:
I noticed we had some tests for the other RPC calls in `client_test.go` but looks like those are more around block data so we can skip this.